spec/fundamentals.mediawiki

```diff
@@ -155,6 +204,7 @@ The goal of this algorithm is to ensure that an individual client cannot obtain
 a deterministic latency advantage over other clients when executing trades.
 Limiting this possibility mitigates advantages gained from front-running,
 spoofing, and other manipulative trading practices.
+Each market will have an epoch duration set by the operator.
```

Comment:
Sentence seems out-of-place here. Perhaps just mention the variable in the `===Market Variables===` section instead.

Reply:
Yeah, I agree.  It's already mentoined in Market Variables, so removing this line.